presto-geospatial-toolkit/src/test/java/com/facebook/presto/geospatial/BenchmarkGeometrySerde.java

```diff
@@ -0,0 +1,404 @@
+        private OGCGeometry complexLineString;
+        private Slice complexLineStringSerialized;
+
+        // MULTILINE STRING
+        private OGCGeometry simpleMultiLineString;
+        private Slice simpleMultiLineStringSerialized;
+        private OGCGeometry complexMultiLineString;
+        private Slice complexMultiLineStringSerialized;
+
+        // POLYGON
+        private OGCGeometry simplePolygon;
+        private Slice simplePolygonSerialized;
+        private OGCGeometry complexPolygon;
+        private Slice complexPolygonSerialized;
+
+        // MULTI POLYGON
+        private OGCGeometry simpleMultiPolygon;
+        private Slice simpleMultiPolygonSerialized;
+        private OGCGeometry complexMultiPolygon;
+        private Slice complexMultiPolygonSerialized;
+
+        // COLLECTION
+        private OGCGeometry simpleGeometryCollection;
+        private Slice simpleGeometryCollectionSerialized;
+        private OGCGeometry complexGeometryCollection;
+        private Slice complexGeometryCollectionSerialized;
+
+        @Setup
+        public void setup()
+                throws IOException
+        {
+            point = fromText(POINT);
+            pointSerialized = serialize(point);
+
+            simpleMultipoint = fromText(MULTIPOINT);
+            simpleMultipointSerialized = serialize(simpleMultipoint);
+            complexMultipoint = fromText(readResource("complex-multipoint.txt"));
+            complexMultipointSerialized = serialize(complexMultipoint);
+
+            simpleLineString = fromText(LINESTRING);
+            simpleLineStringSerialized = serialize(simpleLineString);
+            complexLineString = fromText(readResource("complex-linestring.txt"));
+            complexLineStringSerialized = serialize(complexLineString);
+
+            simpleMultiLineString = fromText(MULTILINESTRING);
+            simpleMultiLineStringSerialized = serialize(simpleMultiLineString);
+            complexMultiLineString = fromText(readResource("complex-multilinestring.txt"));
+            complexMultiLineStringSerialized = serialize(complexMultiLineString);
+
+            simplePolygon = fromText(POLYGON);
+            simplePolygonSerialized = serialize(simplePolygon);
+            complexPolygon = fromText(readResource("complex-polygon.txt"));
+            complexPolygonSerialized = serialize(complexPolygon);
+
+            simpleMultiPolygon = fromText(MULTIPOLYGON);
+            simpleMultiPolygonSerialized = serialize(simpleMultiPolygon);
+            complexMultiPolygon = fromText(readResource("complex-multipolygon.txt"));
+            complexMultiPolygonSerialized = serialize(complexMultiPolygon);
+
+            simpleGeometryCollection = fromText(GEOMETRYCOLLECTION);
+            simpleGeometryCollectionSerialized = serialize(simpleGeometryCollection);
+            complexGeometryCollection = fromText("GEOMETRYCOLLECTION (" + Joiner.on(", ").join(
+                    readResource("complex-multipoint.txt"),
```

Comment:
nit: would it make sense to move `readResource` method and resource names to `BenchmarkData` class to avoid copy-pasting?